src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -321,14 +352,55 @@ private void checkClientServerCompatibility(
   }
 
   @Override
-  public void afterCommand() {
+  public void afterCommand() throws AbruptExitException {
+    IOException failure = null;
+    try {
+      deletedDownloadedInputs();
+    } catch (IOException e) {
+      failure = e;
+    }
+
+    try {
+      closeRpcLogFile();
+    } catch (IOException e) {
+      failure = e;
```

Comment:
in case `closeRpcLogFile();` throws an exception, the one thrown by `deletedDownloadedInputs();` will be lost and ignored. I think that we should attempt to close RpcLogs anyways, so maybe log the first error or collect exceptions to the list and then throw a 'collective' exception, with both failures?

Reply:
I moved the code around so that the error from `deletedDownloadedInputs` takes precedence and added log statements for the rpc log flushing.